packages/orbit-components/src/BadgeList/index.tsx

```diff
@@ -1,12 +1,22 @@
 "use client";
 
 import * as React from "react";
+import cx from "clsx";
 
 import type { Props } from "./types";
+import { getSpacingClasses } from "../common/tailwind";
 
-const BadgeList = ({ children, dataTest, id, ariaLabel }: Props) => {
+const BadgeList = ({ children, dataTest, id, ariaLabel, spacing }: Props) => {
   return (
-    <ul className="m-0 flex flex-col p-0" data-test={dataTest} id={id} aria-label={ariaLabel}>
+    <ul
+      className={cx(
+        "orbit-badge-list m-0 flex flex-col p-0",
+        spacing && getSpacingClasses(spacing, undefined, "column", false),
```

Comment:
Not sure if this is correct or if getSpacingClasses should receive different args